airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalAttemptExecution.java

```diff
@@ -79,7 +98,50 @@ public T get() throws TemporalJobException {
       LOGGER.info("Executing worker wrapper. Airbyte version: {}", EnvConfigs.AIRBYTE_VERSION);
       jobRootDirCreator.accept(jobRoot);
 
-      return execution.apply(jobRoot);
+      final CompletableFuture<T> outputFuture = new CompletableFuture<>();
+
+      final Worker<INPUT, T> worker = workerSupplier.apply(jobRoot);
```

Comment:
I don't see this `jobRoot` argument used anywhere. Can we simplify the worker supplier signature? Am I missing something?

Reply:
this may have been an oversight on my part. if it's not used it can be removed.